package.json

```diff
@@ -24,7 +24,8 @@
     "eyeglass-module"
   ],
   "eyeglass": {
-    "exports": "lib/eyeglass-exports"
+    "exports": "lib/eyeglass-exports",
+    "needs": "*"
```

Comment:
I feel like it should be a warning to have `*` as a version requirement. It's super dangerous to not even require a specific major version.